source/extensions/filters/http/cors/cors_filter.h

```diff
@@ -88,6 +88,7 @@ class CorsFilter : public Http::StreamFilter {
   const std::string& exposeHeaders();
   const std::string& maxAge();
   bool allowCredentials();
+  bool allowPrivateNetworkAccess();
```

Comment:
Where is the implementation of this method?

Reply:
In fact I delete it but not clean all :(